demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -865,6 +865,46 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     );
   }
 
+
+  async getStatsForOutbound(id: string): Promise<void> {
+    const videoElement = document.getElementById(id) as HTMLVideoElement;
+    const stream = videoElement.srcObject as MediaStream;
+    const track = stream.getVideoTracks()[0];
+    let basicReports: {[id: string]: number} =  {};
+
+    let reports = await this.audioVideo.getRTCPeerConnectionStats(track);
+    let duration: number;
+
+    reports.forEach(report => {
+      if (report.type === 'outbound-rtp') {
+        // remained to be calculated
+        this.log(`${id} is bound to ssrc ${report.ssrc}`);
+        basicReports['bitrate'] = report.bytesSent;
+        basicReports['width'] = report.frameWidth;
+        basicReports['height'] = report.frameHeight;
+        basicReports['fps'] = report.framesEncoded;
+        duration = report.timestamp;
+      }
+    });
+
+    await new TimeoutScheduler(1000).start(() => {
```

Comment:
Note that `await` doesn't wait for a second. i.e. `this.getStatsForOutbound` can just be a `void` type..